docs/src/components/header.js

```diff
@@ -174,7 +172,7 @@ export default () => (
     <Root>
       <Wrapper>
         <Flex justifyContent="space-between">
-          <BannerLink href={notifyBaseUrl}>
+          <BannerLink href={notifyUrl('')}>
```

Comment:
Especially here because `notifyUrl('')` doesn't really mean anything whereas `notifyUrl('/')` is more obvious that it links to the root.